change/@ni-nimble-components-db9e952a-a79d-4aa2-a8e9-b3455ee2b620.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Thoughts on calling this a `patch` change. Technically, we are telling clients that they now need to add `appearance-variant="primary"` to buttons put in the `action` slot of the banner. Do we want to consider that a breaking change?

Reply:
Personally, I think the visual difference is subtle enough that we can call this a patch, especially since you plan to update our SLE clients.

Did you consider creating a handler for when the slotted content changes and _forcing_ slotted buttons to one of the prescribed appearances and appearance variant? We already force slotted content to a 24px height, so it doesn't seem like such a strech to also impose other styling requirements (even if via an attribute).

Reply:
I thought briefly about it, but we don't tend to update appearance attributes of slotted elements. I think there's a difference between styling slotted content to place & size it appropriately within another component and changing appearance attributes on it.

I also brought this up in a team meeting, and @rajsite agreed that adding best practice docs rather than mutating the client's element was aligned with how things are expected to work in nimble.

Reply:
I'm ok with `patch` too.